scripts/bindings.py

```diff
@@ -98,27 +102,27 @@ def generic_function(function):
     """
 
     signature = function
-    rtype, function = function.split(' ', 1)
+    rtype, function = function.split(None, 1)
     fname, fargs = function.split('(', 1)
 
     fname = fname.replace('*', '')
     fargs = fargs.replace(');', '')
 
-    text = (
-        '# - {signature}\n'
-        'def {fname}(*args):\n'
-        '    """\n'
-        '    Describe the function.\n'
-        '\n'
-        '    :param args: arguments\n'
-        '    :return: value\n'
-        '    """\n'
-        '\n'
-        '    # TODO\n'
-        '    return clibui.{fname}()\n\n\n'
-    )
+    return '''\
+# - {signature}
+def {fname}(*args):
+    """
+    Describe the function.
+
+    :param args: arguments
+    :return: value
+    """
+
+    # TODO
+    return clibui.{fname}()
+
 
-    return text.format(fname=fname, signature=signature, args=fargs)
+'''.format(fname=fname, signature=signature, args=fargs)
```

Comment:
The proper way to do this, if you really have to, is [`textwrap.dedent`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent).